static/css/perdiem.css

```diff
@@ -390,3 +391,27 @@ li.home-logo {
     margin: 0px;
     padding: 0px;
 }
+
+.album-bio {    
+    margin: 0 auto;
+    padding: 10px;
+}
+
+ol {
```

Comment:
Use the selector `.album-bio > ol` or `.album-bio ol` if you are only trying to select those `<ol>`s.